index.md

```diff
@@ -5,4 +5,83 @@ title: issuehunter
 
 # Introduction
 
-What a wonderful project!
+Issuehunter is a decentralized application that enables Open Source crowdfunding
+on the Ethereum blockchain.
```

Comment:
I'll put a link to the Ethereum website, but I wouldn't like to do it on the first sentence of the introduction to better focus the attention on the content.